src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -32,26 +32,21 @@
 
 namespace Files.App.Views
 {
-	/// <summary>
-	/// The root page of Files
-	/// </summary>
 	public sealed partial class MainPage : Page, INotifyPropertyChanged
 	{
 		private VirtualKeyModifiers currentModifiers = VirtualKeyModifiers.None;
 
 		public IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
+
 		public ICommandManager Commands { get; } = Ioc.Default.GetRequiredService<ICommandManager>();
+
 		public IWindowContext WindowContext { get; } = Ioc.Default.GetRequiredService<IWindowContext>();
 
 		public SidebarViewModel SidebarAdaptiveViewModel = Ioc.Default.GetRequiredService<SidebarViewModel>();
 
 		public AppModel AppModel => App.AppModel;
 
-		public MainPageViewModel ViewModel
-		{
-			get => (MainPageViewModel)DataContext;
-			set => DataContext = value;
-		}
+		public MainPageViewModel ViewModel { get; set; }
```

Comment:
Can you use dependency injection for this? Normally I'd say it's out of scope for the PR but it might be a good idea to take care of it while you're in here.